.travis

```diff
@@ -0,0 +1,16 @@
+language: node_js
```

Comment:
Should this file be named `.travis.yml`?

Reply:
:man_facepalming:

Reply:
Just renamed the file.